### PR TITLE
New Term: NumPy `.linspace()`

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From Target Keyword list.
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/linspace/linspace.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+The `start` and `stop` arguments are required and represent the beginning and end of the interval. They can be numbers or arrays.
+
+`.linspace()` provides the following arguments:
+
+- `start`: The starting point of the sequence.
+- `stop`: The (optionally included) endpoint of the sequence.
+- `num`: The number of values to generate. Defaults to 50.
+- `endpoint`: Boolean flag. If `True` then `stop` is included as the last value. If `False`, `stop` is excluded. Defaults to `True`.
+- `retstep`: Boolean flag. If `True` the result will include the calculated step size between values. Defaults to `False`.
+- `dtype`: The `dtype` of the returned array, if omitted, dtype is inferred from `start` and `stop`. Defaults to `None`.
```
Comment (KTom101) [inline content/numpy/concepts/built-in-functions/terms/linspace/linspace.md:35]:
```suggestion
- `dtype`: The `dtype` of the returned array, if omitted, `dtype` is inferred from `start` and `stop`. Defaults to `None`.
```

File: content/numpy/concepts/built-in-functions/terms/linspace/linspace.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- `axis`: If `start` and `stop` are arrays, specifies on what axis the values will be added. If `0` the axis is added at the beginning. If `-1` it's added at the end. Defaults to `0`.
```
Comment (KTom101) [inline content/numpy/concepts/built-in-functions/terms/linspace/linspace.md:36]:
```suggestion
- `axis`: If `start` and `stop` are arrays, this specifies on what axis the values will be added. If `0` the axis is added at the beginning. If `-1` it's added at the end. Defaults to `0`.
```

File: content/numpy/concepts/built-in-functions/terms/linspace/linspace.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+
+This results in the following output:
+
+```shell
+[10. 12. 14. 16. 18. 20.]
+
+[10.         11.66666667 13.33333333 15.         16.66666667 18.33333333]
+```
```
Comment (KTom101) [inline content/numpy/concepts/built-in-functions/terms/linspace/linspace.md:59]: Can we add another example where `start` and `stop` are arrays?

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski I took a look at this and left a review, thanks!

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski I'll pass this on to a second reviewer!

File: content/numpy/concepts/built-in-functions/terms/linspace/linspace.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+result = numpy.linspace(10, 20, num=6)
+print(result)
+
+result2 = numpy.linspace(10, 20, num=6, endpoint=False)
+print(result2)
+
+result3 = numpy.linspace([1,2,3],[4,5,6], num=6)
+print(result3)
+
```
Comment (Dusch4593) [inline content/numpy/concepts/built-in-functions/terms/linspace/linspace.md:53]:
```suggestion
print(result3)
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Just one suggestion 😄

**Comment by yangc95** [comment]
Taking a look as well, will tag you once I'm fin~

File: content/numpy/concepts/built-in-functions/terms/linspace/linspace.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+The `start` and `stop` arguments are required and represent the beginning and end of the interval. They can be numbers or arrays.
+
+`.linspace()` provides the following arguments:
+
+- `start`: The starting point of the sequence.
+- `stop`: The (optionally included) endpoint of the sequence.
+- `num`: The number of values to generate. Defaults to 50.
+- `endpoint`: Boolean flag. If `True` then `stop` is included as the last value. If `False`, `stop` is excluded. Defaults to `True`.
```
Comment (yangc95) [inline content/numpy/concepts/built-in-functions/terms/linspace/linspace.md:33]:
```suggestion
- `endpoint`: Boolean flag. If `True`, `stop` is included as the last value. If `False`, `stop` is excluded. Defaults to `True`.
```

**Review by yangc95 — COMMENTED** [review]
Minor suggestions @SSwiniarski 🙏

**Comment by SSwiniarski** [comment]
@yangc95, I implemented your changes. @Dusch4593: Your suggestion didn't actually change anything, what is it you wanted to do?

**Comment by yangc95** [comment]
@SSwiniarski thanks! I think @Dusch4593's suggestion was just to remove the extra line in that code block

File: content/numpy/concepts/built-in-functions/terms/linspace/linspace.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+result = numpy.linspace(10, 20, num=6)
+print(result)
+
+result2 = numpy.linspace(10, 20, num=6, endpoint=False)
+print(result2)
+
+result3 = numpy.linspace([1,2,3],[4,5,6], num=6)
+print(result3)
+
```
Comment (yangc95) [inline content/numpy/concepts/built-in-functions/terms/linspace/linspace.md:53]:
```suggestion
```

**Review by yangc95 — APPROVED** [review]

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**